src/App.css

```diff
@@ -36,3 +38,9 @@
     transform: rotate(360deg);
   }
 }
+
+
+.gradient-container {
```

Comment:
in the future, can y'all comment why some choices are inconsistent (e.g., why some of the CSS is in tailwind classnames and some is in the css file)

not bad choices -- just important to document for the future ifl